app/views/duty_deferment_account/duty_deferment_account.scala.html

```diff
@@ -64,10 +64,18 @@
                         msg = messages("cf.account.details.previous-eori", statement.eoriHistory.eori)
                     )
                 }
-                @dutyDeferementHead(statement.groups.head)
 
-                @if(statement.groups.size > 1) {
-                    @accordian(statement.groups.tail, 0)
+                @if(statement.groups.head.monthAndYear.compareTo(model.monthsToDisplay) > 0) {
```

Comment:
We should not write these complex logics inside the view but it already has complex logic hence the entire  logic should be moved out from this view. This should be done in a separate ticket. Better to raise a new task , it would also be a part of live services handover